### PR TITLE
[SHIRO 625] Update 10 minute Tutorial

### PR DESCRIPTION
**Pull request by ddold**
Went with the advice from Brian Demers and implemented a BasicIniEnvironment class that used the IniSecurityManagerFactory class, but its away for the public use

### CODE REVIEW

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/Shiro-pr/25/<h2>Build result: FAILURE</span></h2>[...truncated 224.80 KB...][JENKINS] Archiving /home/jenkins/jenkins-slave/workspace/Shiro-pr/config/core/target/shiro-config-core-1.4.1-SNAPSHOT-javadoc.jar to org.apache.shiro/shiro-config-core/1.4.1-SNAPSHOT/shiro-config-core-1.4.1-SNAPSHOT-javadoc.jar[JENKINS] Archiving /home/jenkins/jenkins-slave/workspace/Shiro-pr/config/core/target/shiro-config-core-1.4.1-SNAPSHOT-sources.jar to org.apache.shiro/shiro-config-core/1.4.1-SNAPSHOT/shiro-config-core-1.4.1-SNAPSHOT-sources.jar[JENKINS] Archiving /home/jenkins/jenkins-slave/workspace/Shiro-pr/test-coverage/pom.xml to org.apache.shiro/shiro-test-coverage/1.4.1-SNAPSHOT/shiro-test-coverage-1.4.1-SNAPSHOT.pom[JENKINS] Archiving /home/jenkins/jenkins-slave/workspace/Shiro-pr/crypto/pom.xml to org.apache.shiro/shiro-crypto/1.4.1-SNAPSHOT/shiro-crypto-1.4.1-SNAPSHOT.pom[JENKINS] Archiving /home/jenkins/jenkins-slave/workspace/Shiro-pr/core/pom.xml to org.apache.shiro/shiro-core/1.4.1-SNAPSHOT/shiro-core-1.4.1-SNAPSHOT.pom[JENKINS] Archiving /home/jenkins/jenkins-slave/workspace/Shiro-pr/samples/aspectj/pom.xml to org.apache.shiro.samples/samples-aspectj/1.4.1-SNAPSHOT/samples-aspectj-1.4.1-SNAPSHOT.pom[JENKINS] Archiving /home/jenkins/jenkins-slave/workspace/Shiro-pr/samples/spring-boot-web/pom.xml to org.apache.shiro.samples/samples-spring-boot-web/1.4.1-SNAPSHOT/samples-spring-boot-web-1.4.1-SNAPSHOT.pom[JENKINS] Archiving /home/jenkins/jenkins-slave/workspace/Shiro-pr/samples/spring-client/pom.xml to org.apache.shiro.samples/samples-spring-client/1.4.1-SNAPSHOT/samples-spring-client-1.4.1-SNAPSHOT.pom[JENKINS] Archiving /home/jenkins/jenkins-slave/workspace/Shiro-pr/tools/hasher/pom.xml to org.apache.shiro.tools/shiro-tools-hasher/1.4.1-SNAPSHOT/shiro-tools-hasher-1.4.1-SNAPSHOT.pom[JENKINS] Archiving /home/jenkins/jenkins-slave/workspace/Shiro-pr/support/spring-boot/spring-boot-starter/pom.xml to org.apache.shiro/shiro-spring-boot-starter/1.4.1-SNAPSHOT/shiro-spring-boot-starter-1.4.1-SNAPSHOT.pom[JENKINS] Archiving /home/jenkins/jenkins-slave/workspace/Shiro-pr/support/jaxrs/pom.xml to org.apache.shiro/shiro-jaxrs/1.4.1-SNAPSHOT/shiro-jaxrs-1.4.1-SNAPSHOT.pom[JENKINS] Archiving /home/jenkins/jenkins-slave/workspace/Shiro-pr/samples/guice/pom.xml to org.apache.shiro.samples/samples-guice/1.4.1-SNAPSHOT/samples-guice-1.4.1-SNAPSHOT.pom[JENKINS] Archiving /home/jenkins/jenkins-slave/workspace/Shiro-pr/crypto/cipher/pom.xml to org.apache.shiro/shiro-crypto-cipher/1.4.1-SNAPSHOT/shiro-crypto-cipher-1.4.1-SNAPSHOT.pom[JENKINS] Archiving /home/jenkins/jenkins-slave/workspace/Shiro-pr/crypto/cipher/target/shiro-crypto-cipher-1.4.1-SNAPSHOT.jar to org.apache.shiro/shiro-crypto-cipher/1.4.1-SNAPSHOT/shiro-crypto-cipher-1.4.1-SNAPSHOT.jar[JENKINS] Archiving /home/jenkins/jenkins-slave/workspace/Shiro-pr/crypto/cipher/target/shiro-crypto-cipher-1.4.1-SNAPSHOT-javadoc.jar to org.apache.shiro/shiro-crypto-cipher/1.4.1-SNAPSHOT/shiro-crypto-cipher-1.4.1-SNAPSHOT-javadoc.jar[JENKINS] Archiving /home/jenkins/jenkins-slave/workspace/Shiro-pr/crypto/cipher/target/shiro-crypto-cipher-1.4.1-SNAPSHOT-sources.jar to org.apache.shiro/shiro-crypto-cipher/1.4.1-SNAPSHOT/shiro-crypto-cipher-1.4.1-SNAPSHOT-sources.jar[JENKINS] Archiving /home/jenkins/jenkins-slave/workspace/Shiro-pr/samples/web/pom.xml to org.apache.shiro.samples/samples-web/1.4.1-SNAPSHOT/samples-web-1.4.1-SNAPSHOT.pom[JENKINS] Archiving /home/jenkins/jenkins-slave/workspace/Shiro-pr/support/servlet-plugin/pom.xml to org.apache.shiro/shiro-servlet-plugin/1.4.1-SNAPSHOT/shiro-servlet-plugin-1.4.1-SNAPSHOT.pom[JENKINS] Archiving /home/jenkins/jenkins-slave/workspace/Shiro-pr/support/spring/pom.xml to org.apache.shiro/shiro-spring/1.4.1-SNAPSHOT/shiro-spring-1.4.1-SNAPSHOT.pom[JENKINS] Archiving /home/jenkins/jenkins-slave/workspace/Shiro-pr/support/guice/pom.xml to org.apache.shiro/shiro-guice/1.4.1-SNAPSHOT/shiro-guice-1.4.1-SNAPSHOT.pomchannel stoppedSetting status of 60b92106aaab3108b96027ea582483f5067f44a4 to FAILURE with url https://builds.apache.org/job/Shiro-pr/25/ and message: 'FAILURE 'Using context: Jenkins: mvn clean install

**Comment by fpapon** [comment]
retest this please

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/Shiro-pr/97/<h2>Build result: FAILURE</span></h2>[...truncated 664.46 KB...][JENKINS] Archiving /home/jenkins/jenkins-slave/workspace/Shiro-pr/core/pom.xml to org.apache.shiro/shiro-core/1.5.0-SNAPSHOT/shiro-core-1.5.0-SNAPSHOT.pom[JENKINS] Archiving /home/jenkins/jenkins-slave/workspace/Shiro-pr/core/target/shiro-core-1.5.0-SNAPSHOT.jar to org.apache.shiro/shiro-core/1.5.0-SNAPSHOT/shiro-core-1.5.0-SNAPSHOT.jar[JENKINS] Archiving /home/jenkins/jenkins-slave/workspace/Shiro-pr/core/target/shiro-core-1.5.0-SNAPSHOT-tests.jar to org.apache.shiro/shiro-core/1.5.0-SNAPSHOT/shiro-core-1.5.0-SNAPSHOT-tests.jar[JENKINS] Archiving /home/jenkins/jenkins-slave/workspace/Shiro-pr/core/target/shiro-core-1.5.0-SNAPSHOT-javadoc.jar to org.apache.shiro/shiro-core/1.5.0-SNAPSHOT/shiro-core-1.5.0-SNAPSHOT-javadoc.jar[JENKINS] Archiving /home/jenkins/jenkins-slave/workspace/Shiro-pr/core/target/shiro-core-1.5.0-SNAPSHOT-sources.jar to org.apache.shiro/shiro-core/1.5.0-SNAPSHOT/shiro-core-1.5.0-SNAPSHOT-sources.jar[JENKINS] Archiving /home/jenkins/jenkins-slave/workspace/Shiro-pr/samples/aspectj/pom.xml to org.apache.shiro.samples/samples-aspectj/1.5.0-SNAPSHOT/samples-aspectj-1.5.0-SNAPSHOT.pom[JENKINS] Archiving /home/jenkins/jenkins-slave/workspace/Shiro-pr/samples/spring-boot-web/pom.xml to org.apache.shiro.samples/samples-spring-boot-web/1.5.0-SNAPSHOT/samples-spring-boot-web-1.5.0-SNAPSHOT.pom[JENKINS] Archiving /home/jenkins/jenkins-slave/workspace/Shiro-pr/samples/spring-client/pom.xml to org.apache.shiro.samples/samples-spring-client/1.5.0-SNAPSHOT/samples-spring-client-1.5.0-SNAPSHOT.pom[JENKINS] Archiving /home/jenkins/jenkins-slave/workspace/Shiro-pr/tools/hasher/pom.xml to org.apache.shiro.tools/shiro-tools-hasher/1.5.0-SNAPSHOT/shiro-tools-hasher-1.5.0-SNAPSHOT.pom[JENKINS] Archiving /home/jenkins/jenkins-slave/workspace/Shiro-pr/support/spring-boot/spring-boot-starter/pom.xml to org.apache.shiro/shiro-spring-boot-starter/1.5.0-SNAPSHOT/shiro-spring-boot-starter-1.5.0-SNAPSHOT.pom[JENKINS] Archiving /home/jenkins/jenkins-slave/workspace/Shiro-pr/support/jaxrs/pom.xml to org.apache.shiro/shiro-jaxrs/1.5.0-SNAPSHOT/shiro-jaxrs-1.5.0-SNAPSHOT.pom[JENKINS] Archiving /home/jenkins/jenkins-slave/workspace/Shiro-pr/samples/guice/pom.xml to org.apache.shiro.samples/samples-guice/1.5.0-SNAPSHOT/samples-guice-1.5.0-SNAPSHOT.pom[JENKINS] Archiving /home/jenkins/jenkins-slave/workspace/Shiro-pr/crypto/cipher/pom.xml to org.apache.shiro/shiro-crypto-cipher/1.5.0-SNAPSHOT/shiro-crypto-cipher-1.5.0-SNAPSHOT.pom[JENKINS] Archiving /home/jenkins/jenkins-slave/workspace/Shiro-pr/crypto/cipher/target/shiro-crypto-cipher-1.5.0-SNAPSHOT.jar to org.apache.shiro/shiro-crypto-cipher/1.5.0-SNAPSHOT/shiro-crypto-cipher-1.5.0-SNAPSHOT.jar[JENKINS] Archiving /home/jenkins/jenkins-slave/workspace/Shiro-pr/crypto/cipher/target/shiro-crypto-cipher-1.5.0-SNAPSHOT-javadoc.jar to org.apache.shiro/shiro-crypto-cipher/1.5.0-SNAPSHOT/shiro-crypto-cipher-1.5.0-SNAPSHOT-javadoc.jar[JENKINS] Archiving /home/jenkins/jenkins-slave/workspace/Shiro-pr/crypto/cipher/target/shiro-crypto-cipher-1.5.0-SNAPSHOT-sources.jar to org.apache.shiro/shiro-crypto-cipher/1.5.0-SNAPSHOT/shiro-crypto-cipher-1.5.0-SNAPSHOT-sources.jar[JENKINS] Archiving /home/jenkins/jenkins-slave/workspace/Shiro-pr/samples/web/pom.xml to org.apache.shiro.samples/samples-web/1.5.0-SNAPSHOT/samples-web-1.5.0-SNAPSHOT.pom[JENKINS] Archiving /home/jenkins/jenkins-slave/workspace/Shiro-pr/support/servlet-plugin/pom.xml to org.apache.shiro/shiro-servlet-plugin/1.5.0-SNAPSHOT/shiro-servlet-plugin-1.5.0-SNAPSHOT.pom[JENKINS] Archiving /home/jenkins/jenkins-slave/workspace/Shiro-pr/support/spring/pom.xml to org.apache.shiro/shiro-spring/1.5.0-SNAPSHOT/shiro-spring-1.5.0-SNAPSHOT.pom[JENKINS] Archiving /home/jenkins/jenkins-slave/workspace/Shiro-pr/support/guice/pom.xml to org.apache.shiro/shiro-guice/1.5.0-SNAPSHOT/shiro-guice-1.5.0-SNAPSHOT.pomchannel stoppedSetting status of 60b92106aaab3108b96027ea582483f5067f44a4 to FAILURE with url https://builds.apache.org/job/Shiro-pr/97/ and message: 'FAILURE 'Using context: Jenkins: mvn clean install

File: core/src/main/java/org/apache/shiro/env/BasicIniEnvironment.java [hunk 1]
```diff
@@ -0,0 +1,22 @@
+package org.apache.shiro.env;
```
Comment (fpapon) [inline core/src/main/java/org/apache/shiro/env/BasicIniEnvironment.java]: @ddold can you add license header please?

Comment (coheigea) [reply core/src/main/java/org/apache/shiro/env/BasicIniEnvironment.java]: @fpapon This PR can be closed IMO. BasicIniEnvironment will be added as part of https://github.com/apache/shiro/pull/157, and the Quickstart no longer uses the IniSecurityManagerFactory

Comment (fpapon) [reply core/src/main/java/org/apache/shiro/env/BasicIniEnvironment.java]: @coheigea you're right, thanks!

**Comment by fpapon** [comment]
Fix by https://github.com/apache/shiro/pull/157

**Pull request closed without merging**